src/redux/missions/missionsSlice.js

```diff
@@ -4,6 +4,7 @@ import { MISSIONS_URL } from '../constants';
 
 const initialState = {
   missionsList: [],
+  myMissions: [],
```

Comment:
-  To refactor the code and utilize only the missionList array while filtering for missions, instead of using the myMissions array
  -  Look at the following code for rocket
<code>

<ul>
        {missionList
          .filter((rocket) => rocket.reserved)
          .map((mission) => (  your code
                     ))}
      </ul